docs/source/guides/contributor/chapters/runners.rst

```diff
@@ -568,3 +568,96 @@ for runnables with kind ``foo``.
 .. literalinclude:: ../../../../../examples/nrunner/runners/avocado-runner-foo
    :language: python
    :linenos:
+
+
+Runners messages
+----------------
+
+Every runner have to send information about the run to the avocado
+core. Those informations are sent by messages which have different
+types based on the information which they are transmitting. Avocado
+understands three main types of messages, the start, running, and
+finish. The start and finish messages are obligatory and every
+runner have to send those. The running messages can transmitting
```

Comment:
I'd use: "The `running` messages can *transmit*" or even better IMO: ""The `running` messages can *contain*"